src/main/java/core/basesyntax/exception/PasswordValidationException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.exception;
+
+public class PasswordValidationException extends RuntimeException {
```

Comment:
move this class out of the separate package please
it should be located in `core.basesyntax` package